### DISCUSSION TITLE
Internal Server Error

### DISCUSSION THREAD

**Discussion by rubensa** [community/community#12975] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm experiencing multiple **Internal Server Error** errors when pushing to my github repos.  Here is an example:

```
git push --force
Enumerating objects: 17, done.
Counting objects: 100% (17/17), done.
Delta compression using up to 8 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (9/9), 740 bytes | 740.00 KiB/s, done.
Total 9 (delta 6), reused 5 (delta 3), pack-reused 0
remote: Resolving deltas: 100% (6/6), completed with 6 local objects.
remote: Internal Server Error
To github.com:rubensa/docker-ubuntu-tini-desktop.git
 ! [remote rejected] 18.04 -> 18.04 (Internal Server Error)
error: failed to push some refs to 'github.com:rubensa/docker-ubuntu-tini-desktop.git'
```

**Comment by rubensa** [comment, marked as answer]
Looks like there is a "massive" degradation of service https://www.githubstatus.com/incidents/sksd097hm0y5

**Reply by BGMP** [reply]
Getting the same issue here

**Reply by JotaCeR** [reply]
There is any news about this? The link says it's solved/updated but I get the same problem since today. And just like one hour ago I was able to push, pull, merge, etc... normally.

**Reply by mbiesiad** [reply]
Here https://www.githubstatus.com/incidents/83lq7ftk19r5 is an info, about 'is experiencing degraded performance' at this moment (17.00 UT+1)

**Reply by mbiesiad** [reply]
Moreover, Downdetector says similar: https://downdetector.com/status/github/
"User reports indicate problems at GitHub"

**Comment by LaborMatthias** [comment]
Cannot push on one repository, but can on another (both private from same company)

**Comment by LaborMatthias** [comment]
Sorry! It finally worked now!

**Comment by JamesArthurHolland** [comment]
I'm getting this issue currently

**Reply by mbcodepl** [reply]
Same on my side.. Any news about this issue?

**Reply by sndour** [reply]
do you have any solutions?

**Reply by peterson-kargo** [reply]
I am also having this issue

**Reply by peterazer90** [reply]
Its working now

**Reply by mbiesiad** [reply]
Downdetector says: https://downdetector.com/status/github/
"User reports indicate problems at GitHub"
and here: 'is experiencing degraded performance' (at this moment)
https://www.githubstatus.com/incidents/83lq7ftk19r5

**Comment by devignesh** [comment]
Thanks in advance, I'm also getting this same issue. Anyone has a solution for this. 

```
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 4 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 457 bytes | 457.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
remote: Internal Server Error
To github.com:devignesh/go-hotel.git
 ! [remote rejected] master -> master (Internal Server Error)
error: failed to push some refs to 'github.com:devignesh/go-hotel.git'
```

**Reply by interglobalmedia** [reply]
I got this issue all of a sudden today. And then was getting other errors when I was trying to logout. And when I finally logged out, couldn't at first log in. And now I can't successfully complete pushing to Github because I get the following:

git push origin main
Enumerating objects: 24, done.
Counting objects: 100% (24/24), done.
Delta compression using up to 8 threads
Compressing objects: 100% (18/18), done.
Writing objects: 100% (18/18), 1.95 KiB | 1000.00 KiB/s, done.
Total 18 (delta 4), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (4/4), completed with 1 local object.
remote: Internal Server Error
To github.com:interglobalmedia/comd2451-sp22-student-grades.git
 ! [remote rejected] main -> main (Internal Server Error)
error: failed to push some refs to 'github.com:interglobalmedia/comd2451-sp22-student-grades.git'

**Comment by inmanturbo** [comment]
I am getting the same error. Unable to push to any repos.  
```bash
! [remote rejected] main -> main (Internal Server Error)
```

**Comment by inmanturbo** [comment]
And it took a few tries to even make the above comment.

**Comment by thamognya** [comment]
I think this issue is recurring again today,

There are many internal server errors (server errors 500), through out all the GitHub services (git actions, pages, [github.community](https://github.community/), and etc.). It seems like there is 'degraded' performance according to [githubstatus.com](https://www.githubstatus.com/), which is affecting every part of Github. 

If you would like, you can monitor this issue in [githubstatus.com](https://www.githubstatus.com/). Currently it degraded in all of the fields.

**Reply by makemefeelgr8** [reply]
Not this again... It's been 2 days since guys from my team and I were able to actually push something. Even thinking about upcoming merge makes me sick.

**Comment by maaspj2022** [comment]
same issue . looking like all is well. but when clone . is old ver
Enumerating objects: 11, done.
Counting objects: 100% (11/11), done.
Delta compression using up to 8 threads
Compressing objects: 100% (6/6), done.
Writing objects: 100% (6/6), 595 bytes | 595.00 KiB/s, done.
Total 6 (delta 5), reused 0 (delta 0), pack-reused 0
remote: Analyzing objects... (6/6) (55 ms)
remote: Storing packfile... done (37 ms)
remote: Storing index... done (38 ms)
To github/git.git 
   b44e209..8867cdf  main -> main
